sklearn/compose/_column_transformer.py

```diff
@@ -233,7 +233,7 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
     >>> # "documents" is a string which configures ColumnTransformer to
     >>> # pass the documents column as a 1d array to the FeatureHasher
     >>> ct = ColumnTransformer(
-    ...     [("text_preprocess", FeatureHasher(input_type="string"), "documents"),
+    ...     [("text_preprocess", CountVectorizer(), "documents"),
```

Comment:
There is one more mention of `FeatureHasher` two lines up:

https://github.com/scikit-learn/scikit-learn/blob/7e18b68f90c69cabcfc5858228260f50ad7d0b24/sklearn/compose/_column_transformer.py#L234

Can you update that too?

Reply:
Oopsy, sorry about that!